test/integration/targets/launchd/README.md

```diff
@@ -0,0 +1,20 @@
+# Launchd integration tests
```

Comment:
Existing documentation covers how to run integration tests, so this file isn't needed.

See: https://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html